cli/command/version/version.go

```diff
@@ -38,7 +40,7 @@ var versionTemplate = `amp:
 amplifier:      {{if .IsConnected}}
  Version:       {{.Server.Version}}
  Build:         {{.Server.Build}}
- Address:       {{.Server.Address}}
+ Port:          {{.Server.Port}}
```

Comment:
Not sure `Port` is needed. If we can connect to the service from the cli, then we already know the port. It could possibly be interesting to know if the port is different because of port mapping, but that probably isn't really needed (and potentially confusing). Probably better to remove it and add later if we realize we do need it.